packages/commonwealth/client/scripts/controllers/chain/ethereum/NftAdapter.ts

```diff
@@ -36,16 +36,18 @@ export default class Nft extends Ethereum implements ITokenAdapter {
   // Extensions of Ethereum
   constructor(meta: ChainInfo, app: IApp) {
     super(meta, app);
-    this.contractAddress = meta.address;
+    // iterate through selectedChain.Contracts for the Nft type and return the address
+    const nftContracts = app.contracts.getByType('erc721');
```

Comment:
can we have a list or enum of types somewhere

Reply:
👍